src/components/Dashboard/Dashboard.module.scss

```diff
@@ -46,11 +47,10 @@ $color-black: #000;
     .container {
         display: flex;
         flex-direction: column;
-        width: 100%;
         align-items: center;
     }
     .searchBar {
-        width: 100%;
+        width: 95%;
```

Comment:
Is this for spacing? This is not how you're supposed to give space. You should be using padding on the container which will give space around the container.

Reply:
could you elaborate?

Reply:
If I already have a width of 100% set why not reduce just that than writing an extra line

Reply:
(one of the ways to get to the solution) 
could you suggest official docs that say that using % for spacing is not good

Reply:
Yet again there are multiple methods to reach the same solution, If you could provide me with a strong official documentation that says this in written I will happily change the code

Reply:
Spacing in the world of CSS refers to padding, margins, gaps. https://web.dev/learn/css/spacing/

No one uses width. The use case of width is to specify dimensions of a box. That's the use case. 

As for this `"You "SHOULD" not use % or change width to provide spacing"` there's no documentation that specifies this, but it's a logical understanding.

Reply:
> As for this "You "SHOULD" not use % or change width to provide spacing" there's no documentation that specifies this, but it's a logical understanding.

I am kind of dumb on these parts so if any doc says I will follow that, every developer employee's their own way to provide spacing